packages/neuron-wallet/tests/setupAndTeardown/index.ts

```diff
@@ -1,7 +1,6 @@
 import initDB from "../../src/database/chain/ormconfig"
 import { getConnection } from 'typeorm'
-import transactions from './transactions'
-import accounts from './accounts'
+// import accounts from './accounts.fixture'
```

Comment:
Remove this comment if it is dead code.

Reply:
I think they've been removed in the following commits.